### PR TITLE
Hello microverse corrections

### PR DESCRIPTION
**Pull request by Chimoneg27**
### Final Hello-Microverse corrections

- Removed test.md no longer exists. 
- README.md has been properly updated
- README.md  default texts have been removed
- styles.css styling errors fixed with the help of linters. Unnecessary whites spaces removed 
- .github file has been removed from .gitignore file
- html and CSS linter files have been added
- https://github.com/Chimoneg27/Hello-World/pull/1 (link to my very first pull request I made ) 
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,92 @@
...
+
+👤 **Garvin**
+
+- GitHub: [@Chimoneg27](https://github.com/Chimoneg27)
+- Twitter: [@Chimoneg_27](https://twitter.com/Chimoneg_27)
+- LinkedIn: [Garvin Chimone](https://www.linkedin.com/in/garvin-chimone-37208924a/)
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
Comment (souad988) [inline README.md]: Great work @Chimoneg27 , just a small last thing to fix before the approval, kindly remove the default text under each section to make your work look clean and professional

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Chimoneg27 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

212385281-4797c58a-d358-423a-aa7c-832c22203865

 ## Highlights
✔Use of a correct github workflow
✔Use of Microverse README template 
✔Great efforts

 ## Required Changes ♻️
_Check the comments under the review._
_ Also consider using a descriptive PR title that reflects the work done in this feature exp: setup html/css linters ...
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Chimoneg27) [reply README.md]: @souad988. It's all fixed. You may check the code

**Review by V-Blaze — APPROVED** [review]
Hi @Chimoneg27  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-accepted


## Highlights

- Readme Looks professional💯
-  Great communication with the reviewer👍🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**